app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -320,6 +321,7 @@ class SuggestedEditsTasksFragment : Fragment() {
         disabledStatesView.setIPBlocked()
         disabledStatesView.visibility = VISIBLE
         UserContributionFunnel.get().logIpBlock()
+        UserContributionEvent().logIpBlock()
```

Comment:
If we use `UserContributionEvent()` directly, doesn't that mean we have to create an instance for every action? Does that take extra memory usages?

Reply:
That was the problem with the previous way of implementation @cooltey . When only one object is used [as in singleton], the same object was queued multiple times. So this is the workaround..

Reply:
I see. Make sense.